src/snapred/ui/workflow/WorkflowBuilder.py

```diff
@@ -1,14 +1,25 @@
+from snapred.meta.Config import Config
 from snapred.ui.model.WorkflowNodeModel import WorkflowNodeModel
 from snapred.ui.widget.Workflow import Workflow
 
 
 class WorkflowBuilder:
-    def __init__(self, *, startLambda=None, iterateLambda=None, resetLambda=None, cancelLambda=None, parent=None):
+    def __init__(
+        self,
+        *,
+        startLambda=None,
+        iterateLambda=None,
+        resetLambda=None,
+        cancelLambda=None,
+        parent=None,
+        completionMessageLambda=lambda: Config["ui.default.workflow.completionMessage"],
```

Comment:
I feel like I could reduce the number of lambdas by just passing the WorkflowImplementer itself.